src/v/redpanda/admin_server.cc

```diff
@@ -349,6 +349,45 @@ void admin_server::log_request(
       req.get_url());
 }
 
+void admin_server::log_exception(
+  const ss::sstring& url,
+  const request_auth_result& auth_state,
+  std::exception_ptr eptr) const {
+    using http_status = ss::httpd::reply::status_type;
+    using http_status_ut = std::underlying_type_t<http_status>;
+    const auto log_ex = [&](int status) {
```

Comment:
nit: rather than having a status with an implicit ">0 is valid" behavior, would be nicer to make it a std::optional so that it's obvious what's going on when reading the function declaration.